pkg/controllers/nodeclaim/disruption/expiration_test.go

```diff
@@ -42,11 +42,13 @@ var _ = Describe("Expiration", func() {
 			ObjectMeta: metav1.ObjectMeta{
 				Labels: map[string]string{v1.NodePoolLabelKey: nodePool.Name},
 			},
+			Spec: v1.NodeClaimSpec{
+				ExpireAfter: v1.NillableDuration{Duration: lo.ToPtr(time.Second * 30)},
+			},
 		})
 	})
 	Context("Metrics", func() {
 		It("should fire a karpenter_nodeclaims_disrupted metric when expired", func() {
-			nodePool.Spec.Disruption.ExpireAfter.Duration = lo.ToPtr(time.Second * 30)
```

Comment:
nit. Can we now just remove the nodePool from this test file?

Reply:
good call. I kept it in for some of the metrics tests, but removed it elsewhere